airbyte-integrations/bases/base-normalization/setup.py

```diff
@@ -31,7 +31,7 @@
     author_email="contact@airbyte.io",
     url="https://github.com/airbytehq/airbyte",
     packages=setuptools.find_packages(),
-    install_requires=["airbyte-protocol", "dbt>=0.18.1", "pyyaml"],
+    install_requires=["airbyte-protocol", "dbt==0.19.0", "pyyaml"],
```

Comment:
Upgrading dbt to version 0.19.0 (and dbt-utils accordingly) to solve issues with postgres naming: https://github.com/airbytehq/airbyte/issues/1750